### PR TITLE
Add strings for tracker blocking callouts

### PR DESCRIPTION
**Pull request by Vinnl**
These are for new callouts on the landing page and the Premium
interstitial to promote tracker removal and promotional email
blocking.

image

and

image

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -321,12 +323,17 @@ premium-promo-perks-headline = Why upgrade to { -brand-name-firefox-relay-premiu
...
 premium-promo-perks-perk-custom-domain-body-2 = With a custom subdomain, you can create masks that are easier than ever to remember and share. Need one for restaurant reservations? Use one like food@mydomain.mozmail.com — No need to create the mask beforehand.
 premium-promo-perks-perk-dashboard-headline-2 = Control your masks from the dashboard
 premium-promo-perks-perk-dashboard-body-2 = Manage all your email masks in the easy-to-use dashboard: if you find that one receives unwanted messages, you can block those messages from reaching your inbox.
+premium-promo-perks-perk-block-promotionals-headline = Block promotional emails
+premium-promo-perks-perk-block-promotionals-body = With { -brand-name-relay-premium }, you can block promotional emails from reaching your inbox while still receiving emails like receipts or shipping information.
+premium-promo-perks-perk-tracker-blocking-headline = Remove email trackers
+premium-promo-perks-perk-tracker-blocking-body = Now { -brand-name-relay } can help you stop email tracking—your email masks will remove common email trackers from any emails forwarded to you, helping you stay invisible to trackers and advertisers.
```
Comment (peiying2) [inline en/app.ftl:336]: Should there be spaces before and after the hyphen?

**Review by peiying2 — COMMENTED** [review]
Overall the PR looks good. One comment on how the hyphen is used.

File: en/app.ftl (hunk 1 above)
Comment (Vinnl) [reply en/app.ftl:336]: I'll double-check.

Comment (Vinnl) [reply en/app.ftl]: I quote:

> yes, good catch, thank you

Fixed now!

**Review by peiying2 — APPROVED** [review]
Thanks for the update. Really appreciate sharing the mockups to provide better context.

**Pull request merged**